### PR TITLE
RF-5311 Add circle 2 config

### PR DESCRIPTION
**Pull request by kruszczynski**

### CODE REVIEW

File: .circleci/setup-heroku.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```
Comment (jbarber) [inline .circleci/setup-heroku.sh]: I think this script is in the `heroku-deploy` docker image, so you don't need to include it here.

Comment (jbarber) [reply .circleci/setup-heroku.sh]: (I think this is an innovation that might not have been back ported to all repos)

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+jobs:
+  lint:
+    docker:
+      - image: circleci/node:6.14
+        environment:
+          MOCHA_FILE: $CIRCLE_TEST_REPORTS/junit/test-results.xml
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-tester-chrome-{{ checksum "package.json" }}
```
Comment (jbarber) [inline .circleci/config.yml]: Should be `v1-tester-chrome-extension` to fit in with other repos...

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+  upload_prod:
+    docker:
+      - image: circleci/node:6.14
+        environment:
+          MOCHA_FILE: $CIRCLE_TEST_REPORTS/junit/test-results.xml
+    steps:
+      - checkout
+      - run:
+          name: Install awscli
+          command: sudo apt-get -y -qq install awscli
```
Comment (jbarber) [inline .circleci/config.yml]: I think there's a docker image for this?

File: .circleci/setup-heroku.sh (hunk 1 above)
Comment (kruszczynski) [reply .circleci/setup-heroku.sh]: Done

File: .circleci/config.yml (hunk 3 above)
Comment (kruszczynski) [reply .circleci/config.yml]: That's a possible improvement - we could compile JS in one container and then upload in another, but I don't want to spend more time on this at this moment

**Pull request merged**